csharp/extractor/Semmle.Extraction.CSharp.DependencyFetching/DependencyManager.cs

```diff
@@ -60,20 +58,21 @@ public DependencyManager(string srcDir, IDependencyOptions options, ILogger logg
             this.progressMonitor.FindingFiles(srcDir);
 
             packageDirectory = new TemporaryDirectory(ComputeTempDirectory(sourceDir.FullName));
-            var allFiles = GetAllFiles().ToList();
-            var smallFiles = allFiles.SelectSmallFiles(progressMonitor).SelectFileNames();
-            this.fileContent = new FileContent(progressMonitor, smallFiles);
-            this.allSources = allFiles.SelectFileNamesByExtension(".cs").ToList();
-            var allProjects = allFiles.SelectFileNamesByExtension(".csproj");
+            tempWorkingDirectory = new TemporaryDirectory(GetTemporaryWorkingDirectory());
+
+            var allFiles = GetAllFiles();
+            var binaryFileExtensions = new HashSet<string>(new[] { ".dll", ".exe" }); // TODO: add more binary file extensions.
```

Comment:
Yes, good idea.
Maybe an easy win could also be exclude compressed files (and also uncompressed trap files even though they are text files) (Not for this PR).